### ISSUE TITLE
Add semicolon as email separator

### ISSUE THREAD

**Issue by Dalmirog** [OctopusDeploy/Issues#1555]
Currently we only support the comma for this. Adding semicolon support makes sense, as it is the most used character for email separation.

**Comment by Dalmirog** [comment]
Note to self to update doc once this is done

http://docs.octopusdeploy.com/display/OD/Email+notifications

**Comment by PaulStovell** [comment]
Also, add logging to make it clear to the user what those email address fields evaluate to before we try to add them

**Comment by zentron** [comment]
merged into release *build-3.0.0.1118*
Email addresses now accept semicolon or comma as separator.

**Comment by alfhenrik** [comment]
:tada: :cake: :grin:
